test-requirements.txt

```diff
@@ -1 +1,2 @@
 freezegun==1.2.2
+mock
```

Comment:
this is not needed. Simply replace `import mock` with `from unittest import mock` :wink: